critiquebrainz/frontend/external/musicbrainz_db/entities.py

```diff
@@ -0,0 +1,46 @@
+from critiquebrainz.frontend.external.musicbrainz_db.release_group import fetch_multiple_release_groups, get_release_group_by_id
+from critiquebrainz.frontend.external.musicbrainz_db.place import fetch_multiple_places, get_place_by_id
+from critiquebrainz.frontend.external.musicbrainz_db.event import fetch_multiple_events, get_event_by_id
+
+
+def get_multiple_entities(entities):
+    """Fetch multiple entities using their mbids.
+
+    Args:
+        entites: List of tuples containing the entity_id and the entity_type.
+
+    Returns:
+        Dictionary containing the entities keyed by their mbid.
```

Comment:
Can you also mention what the "entity" in the output actually is? Perhaps describing the output for each type is unnecessary, but at least mentioning what kind of information is expected there and what that depends on might be nice.